make/devices.cmake

```diff
@@ -565,7 +565,7 @@ endfunction()
 
 set(VPR_BASE_ARGS
     --min_route_chan_width_hint 100
-    --verbose_sweep on
+#    --verbose_sweep on
```

Comment:
This was changes in master, can rebase and remove this

Reply:
Yeah, my mistake. I did the rebase.